go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -732,20 +746,37 @@ func (oc *DefaultNetworkController) getEgressFirewallACLDbIDs(namespace string,
 		})
 }
 
-func (oc *DefaultNetworkController) updateEgressFirewallForNode(oldNode, newNode *kapi.Node) error {
+func (oc *DefaultNetworkController) newEFNodeController(nodeInformer coreinformers.NodeInformer) controller.Controller {
+	controllerConfig := &controller.Config[kapi.Node]{
+		RateLimiter:    workqueue.NewItemFastSlowRateLimiter(time.Second, 5*time.Second, 5),
```

Comment:
Can you explain the choice of non default rate limiter in a comment?

Reply:
I guess I am wrong and egressip etc where the first controllers to introduce it. I will have to live with the doubt.